linux_os/guide/services/ssh/ssh_server/sshd_use_approved_kex_ordered_stig/tests/correct_scrambled.fail.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-# platform = multi_platform_ol,multi_platform_rhel,multi_platform_sle
+# platform = multi_platform_ol,multi_platform_rhel,multi_platform_sle,multi_platform_ubuntu
```

Comment:
Now that you are touching this file, could you fix the error a few lines below?

it has:
`KEX_ALGOS_SCRAMBLED=$(echo ${KEY_ALGOS_ARR[0]},${KEY_ALGOS_ARR[1]},$(echo ${KEY_ALGOS_ARR[@]:2} | tr " " ","))`
but should be:
`KEX_ALGOS_SCRAMBLED=$(echo ${KEX_ALGOS_ARR[0]},${KEX_ALGOS_ARR[1]},$(echo ${KEX_ALGOS_ARR[@]:2} | tr " " ","))`
basically change `KEY_ALGOS_ARR` -> `KEX_ALGOS_ARR`

Reply:
done, but I also had to make a small fix to the logic as the order was not being scrambled at all.
thanks!